data/json/items/armor.json

```diff
@@ -12247,14 +12247,14 @@
         "material" : ["cotton", "leather"],
         "volume" : 10,
         "cutting" : 0,
-        "warmth" : 35,
+        "warmth" : 40,
         "phase" : "solid",
         "environmental_protection" : 4,
-        "encumbrance" : 3,
+        "encumbrance" : 2,
         "bashing" : 0,
         "flags" : ["VARSIZE", "WATERPROOF"],
         "coverage" : 95,
-        "material_thickness" : 4
+        "material_thickness" : 5
```

Comment:
Originally the cowl had 4 thickness, but the suit had 8. I wanted the cowl to have the same protection as the suit, since they are parts of one "set". So I made both 5.